addons/beehave/debug/debugger_tab.gd

```diff
@@ -41,16 +41,16 @@ func _ready() -> void:
 	button.pressed.connect(func(): make_floating.emit())
 	button.tooltip_text = "Make floating"
 	button.focus_mode = Control.FOCUS_NONE
-	graph.get_zoom_hbox().add_child(button)
+	graph.get_menu_hbox().add_child(button)
```

Comment:
something like this might be better for backwards compatibility. Or even introduce a new method below:
```gdscript
func get_box() -> Node2D:
   if graph.has_method("get_menu_hbox"):
      # Godot 4.0+
      return graph.get_menu_hbox()
   else:
      # Godot 4.2+
      return graph.get_zoom_hbox()
```
```suggestion
        get_box().add_child(button)
```